src/helpers/MarkdownHelpers.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Groups an array of changes by their 'type' property
+ * @param object The input object
+ * @returns The grouped object
+ */
+import {MarkdownSubtype} from '../types';
+import convertToYAML from './output/convertToYAML';
+
+export function groupChangesByType(object: any): { string: [{ path: string, any: any }] } {
+  return object.reduce((objectsByKeyValue: { [x: string]: any; }, obj: { [x: string]: any; }) => {
+    const value = obj['type'];
+    // eslint-disable-next-line security/detect-object-injection
+    objectsByKeyValue[value] = (objectsByKeyValue[value] || []).concat(obj);
+    return objectsByKeyValue;
+  }, {});
+}
+
+/**
+ * Sets the first letter of a string to uppercase
+ * @param s The input string
+ * @returns The string with the first letter capitalised
+ */
+export function capitaliseFirstLetter(s: string): string {
+  return s[0].toUpperCase() + s.slice(1);
+}
+
+/**
+ * Generates the Markdown list items for a single change
+ * @param change The object describing the change
+ * @param markdownSubtype the format to display the dropdown data in
+ * @returns The Markdown list describing the change
+ */
+export function generateMarkdownForChange(change: { path: string, any: any }, markdownSubtype: MarkdownSubtype): any {
```

Comment:
I think these function types(ex. `change: { path: string, any: any }, markdownSubtype: MarkdownSubtype): any`) should be put inside the `types.ts` file. What do you think?